README.md

```diff
@@ -67,9 +67,6 @@ A library providing a collection of utilities for adaptive layouts.
 ### 🗜 [Test Harness](./testharness/)
 Utilities for testing Compose layouts.
 
-### 📐 [Insets](./insets/) (Deprecated)
```

Comment:
It might be worth leaving this in the README as "Deprecated and Removed" with a link to the guide of developer.android.com.